.rubocop.yml

```diff
@@ -1,15 +1,11 @@
 # use the shared YaST defaults
 inherit_from:
-  /usr/share/YaST2/data/devtools/data/rubocop_yast_style.yml
```

Comment:
If you're going to fix the Rubocop problems in a different PR, maybe you can drop this commit from this PR.

Reply:
The rubocop config file was broken. I think it can still be here. Otherwise, all the real rubocop issues would not be exposed at all.